contrib/clemux/docker/debile-slave/Dockerfile

```diff
@@ -15,7 +15,16 @@ COPY *.deb /tmp/
 
 WORKDIR /tmp
 
-RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml sbuild python-schroot
+RUN apt-get update && apt-get install -y python python-dput dput-ng python-firehose adduser python-debian python-requests python-yaml python-schroot wget
+
+# Use a custom version of sbuild written by Léo Cavaillé that adds an
+# option --use-schroot-session, which enables starting sbuild with an
+# existing schroot session.
+RUN wget https://dimjasevic.net/marko/debile/libsbuild-perl_0.69.0-2_all.deb
```

Comment:
I agree that we should try to have the patch merged into sbuild upstream. However, that could take some time.

In the meantime, I'd rather have the Dockerfile download sbuild's source, apply the patch and build it. But Sylvestre will have the final word on that :)